containers/ddev-dbserver/Makefile

```diff
@@ -46,7 +46,7 @@ build: container
 
 # The format here is majorversion:pinnedimageversion
 # This allows to pin the version (as currently required with 8.0.19
-MARIADB_VERSIONS=5.5:5.5 10.0:10.0 10.1:10.1 10.2:10.2 10.3:10.3 10.4:10.4 10.5:10.5
+MARIADB_VERSIONS=10.1:10.1 10.2:10.2 10.3:10.3 10.4:10.4 10.5:10.5
```

Comment:
This is just a quick fix to get the images to build. If 5.5 and 10.0 support are required moving forward, we can consider adding some conditionals to the build script.

Reply:
We do need 5.5 and 10.0 I think. So conditionals will be the ticket.